packages/web-shared/components/VideoPlayer/VideoPlayer.js

```diff
@@ -39,7 +39,7 @@ function VideoPlayer(props = {}) {
     (video) => video.sources?.length && video.sources[0].uri?.includes('.m3u8')
   );
   const youtubeMedia = props.parentNode?.videos?.find(
-    (video) => video.sources?.length && video.sources[0].uri?.includes('youtube.com')
```

Comment:
https://github.com/ApollosProject/apollos-platforms/blob/master/templates/mobile/new-src/components/MediaPlayer/YoutubePlayer.tsx#L24-L26

Here's the regex we use on the mobile app if you want to duplicate